test/integration/integration_admin_test.cc

```diff
@@ -138,8 +138,13 @@ TEST_P(IntegrationAdminTest, Admin) {
   EXPECT_THAT(response->body(), HasSubstr("admin commands are:"));
 
   EXPECT_EQ("200", request("admin", "GET", "/", response));
+#ifdef ENVOY_ADMIN_HTML
```

Comment:
note: in other cases I did not need a `-D` for this, but it seemed painful to tweak this particular test without it.

I have this gut feel that this define is costly from a build perspective since it is passed on every compile-command, even though it's needed only by one test.

LMK if you think I should re-work this somehow.